test/e2e/flask/solana/common-solana.ts

```diff
@@ -7,31 +7,216 @@ import FixtureBuilder from '../../fixture-builder';
 import { ACCOUNT_TYPE } from '../../constants';
 import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
 
-const SOLANA_URL_REGEX = /^https:\/\/.*\.solana.*/u;
-
+const SOLANA_URL_REGEX = /.*/u;
+// const SOLANA_RPC_PROVIDER = 'https://api.devnet.solana.com/';
```

Comment:
Non blocking, but to consider for the next PR, the regex pattern above seems a very permissive, I'm wondering why aren't we using the RPC URL for Solana that's currently used

Reply:
It is very permissive, indeed, but it's mainly for the reason I mentioned in your other comment. Once we have a proper DIN/Infura endpoint, this will change